codegen/src/next/mod.rs

```diff
@@ -124,7 +126,23 @@ impl Endpoint {
                         has_err = true;
                     }
                     if let Expr::Path(path) = value {
-                        method = Some(path.path.segments[0].ident.clone());
+                        let method_ident = Some(path.path.segments[0].ident.clone());
+
+                        // Unwrap is safe because we just set the value
+                        match method_ident.as_ref().unwrap().to_string().as_str() {
+                            "get" | "post" | "delete" | "put" | "options" | "head" | "trace"
+                            | "patch" => {
+                                method = method_ident;
+                            }
```

Comment:
```suggestion
                        let method_ident = path.path.segments[0].ident.clone();

                        match method_ident.as_ref().to_string().as_str() {
                            "get" | "post" | "delete" | "put" | "options" | "head" | "trace"
                            | "patch" => {
                                method = Some(method_ident);
                            }
```
So that we don't wrap to unwrap again - ie moving the wrap down